manager/controllers/system_advisories_export_test.go

```diff
@@ -41,3 +41,14 @@ func TestSystemAdvisoriesExportCSV(t *testing.T) {
 	assert.Equal(t, "id,description,public_date,synopsis,advisory_type,severity,cve_count", lines[0])
 	assert.Equal(t, "RH-1,adv-1-des,2016-09-22T16:00:00Z,adv-1-syn,1,,0", lines[1])
 }
+
+func TestUnknownSystemAdvisoriesExport(t *testing.T) {
```

Comment:
Please keep test methods name convention "specific part to the end" - here TestSystemAdvisoriesExportUnknown. It makes it easiers to find tests for the same endpoint when printed in logs. Also for other tests.